Tests/Clappr_Tests/Classes/Plugin/Playback/AVFoundationPlaybackTests.swift

```diff
@@ -928,6 +928,22 @@ class AVFoundationPlaybackTests: QuickSpec {
                 }
             }
 
+            context("when onFailedToPlayToEndTime") {
+                it("dispatch error") {
+                    let playback = AVFoundationPlaybackMock(options: [:])
```

Comment:
What you think of using the real class here instead of the mock class? if I change the real method this test still pass.

Reply:
Done